packages/events-api/src/adapter.ts

```diff
@@ -119,23 +120,18 @@ export class SlackEventAdapter extends EventEmitter {
 
   /**
    * Returns a middleware-compatible adapter.
-   * @param middlewareOptions - (UNUSED)
    */
-  public expressMiddleware(
-    middlewareOptions: object = {},
-  ): (req: IncomingMessage, res: ServerResponse, next: () => void) => void {
-    const requestListener = this.requestListener(middlewareOptions);
+  public expressMiddleware(): (req: IncomingMessage, res: ServerResponse, next: () => void) => void {
```

Comment:
Erm, this begs the question: what's the difference between `expressMiddleware` & `requestListener`?

Also, changing the return type of this method to a function that accepts only `req` and `res` means that the returned function only has a `length` of 2 (instead of the current 3). `express` *does* read this number (last I checked) when determining whether a middleware should be run serially or async, and I'm afraid of downstream apps somehow depending on this.

Reply:
The only difference should be the arity (number of arguments expected) of the function. `expressMiddleware` should have an arity of 3, which is what the `RequestHandler` type has. `requestListener` should have an arity of 2, which is what the `RequestListener` type has.